### ISSUE TITLE
Please update this to 1.18.2. Also MAJOR ISSUE: No PepeLaugh

### ISSUE THREAD

**Issue by TheShinPin** [Algid/FabricEmotes#2]
:)

**Comment by Algid** [comment]
PepeHands I'll try to find some time to do an update over the weekend to update the mod & fix that game breaking bug

**Comment by TheShinPin** [comment]
Also, you should just add BetterTTV support, that way you could just choose which BetterTTV emotes you want active. Also, I think it would be cool if this was also server sided and servers could choose which emotes they want active and which they don't, and since it uses BetterTTV adding custom server emotes would be a lot easier. Also an option to not have to use : and just type the word. PepeLaugh instead of :PepeLaugh: for example.

**Comment by Algid** [comment]
I updated this mod the other weekend to 1.18.2 and also added PepeLaugh - in terms of the above suggestions I plan to work on adding custom emotes although I am not quite sure exactly how this will work yet or exactly when I will find the time. I do agree that if emotes could be configured on the server that would be nice, although more complex & I do wish to keep support for clientside only install of this mod but it's something I'll consider. I also plan to look into using BTTV or a Discord bot for custom emotes in the future.

I will close the issue as the original issue has been solved & have made note of these ideas.
